misk-redis-testing/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -380,14 +306,13 @@ class FakeRedisTest {
   @Test fun hIncrBySupportsExpiry() {
     // Setup
     redis.hset("foo", "bar", "1".encodeUtf8())
-    redis.expire("foo", 1)
 
     // Assert that the expiry hasn't taken affect yet
     redis.hincrBy("foo", "bar", 2)
     assertEquals("3", redis.hget("foo", "bar")?.utf8())
 
     // Exercise
-    clock.add(Duration.ofSeconds(1))
+    redis.expire("foo", -1)
```

Comment:
I did this in as many places as possible where we were just trying to validate behaviour after a key has been expired/deleted.

This is because when we test RealRedis, we can't manipulate the clock.

This is _technically_ a behaviour change, because Redis docs indicate that when you call EXPIRE (and friends) with negative arguments actually the key is _deleted_ instead of expired, but it's not meaningfully different for our usages.